ecosystem-tool-user-guide.md

```diff
@@ -1,272 +1,86 @@
 
-[TiDB Data Migration](https://pingcap.com/docs/tidb-data-migration/stable/) (DM) is an integrated data replication task management platform that supports the full data migration and the incremental data migration from MySQL/MariaDB into TiDB. It can help to reduce the operations cost and simplify the troubleshooting process. 
+## Back up and restore
 
-For the full data migration, it uses an embedded Loader and an embedded Mydumper. For the incremental data migration, it uses Syncer as its kernel.
-
-The following are the basics of DM:
-
-- Input data source: MySQL/MariaDB master host/port
-- Supported TiDB versions: all versions
-- Kubernetes support: No, under development (the estimated time is 2020 Q2)
-
-## Data export (backup)
-
-### Full data export tools
-
-#### Mydumper
-
-[Mydumper](/mydumper-overview.md) is a tool to create a logical full backup for TiDB.
-
-The following are the basics of Mydumper:
-
-- Input/Output
-    - Input: TiDB/MySQL host:port 
-    - Output: schema and insert statements file
-- Supported TiDB versions: all versions
-- Kubernetes support: Yes. See [Backup and Restore](https://pingcap.com/docs/tidb-in-kubernetes/stable/backup-and-restore-using-helm-charts/) for details.
-
-#### BR (beta)
-
-[BR](/br/backup-and-restore-tool.md) (Backup & Restore) is a command-line tool for distributed backup and restoration of the TiDB cluster data. Compared with Mydumper/loader, BR is more suitable for scenarios of huge data volume.
+[BR](/br/backup-and-restore-tool.md) (Backup & Restore) is a command-line tool for distributed backup and restoration of the TiDB cluster data. BR can effectively back up and restore TiDB clusters of huge data volume.
 
 The following are the basics of BR:
 
-- Input/Output
-    - Input: TiDB cluster
-    - Output: Full backup file
-- Supported TiDB versions: v3.1 or v4.0
-- Kubernetes support: Yes. The document is WIP.
-
-### Incremental data export tools
+- Input and output data source: SST + `backupmeta` file
+- Supported TiDB versions: v3.1 and v4.0
+- Kubernetes support: Yes. See [Back up Data to S3-Compatible Storage Using BR](https://pingcap.com/docs/tidb-in-kubernetes/stable/backup-to-aws-s3-using-br/) and [Restore Data from S3-Compatible Storage Using BR](https://pingcap.com/docs/tidb-in-kubernetes/stable/restore-from-aws-s3-using-br/) for details.
 
-#### TiDB Binlog
+## Incremental log replication
 
-[TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) is a system that collects binlog for TiDB clusters and provides tools for near real-time sync and backup.
+[TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) is a tool that collects binlog for TiDB clusters and provides near real-time sync and backup. It can be used for incremental data replication between TiDB clusters, such as making a TiDB cluster the slave cluster of another TiDB cluster.
```

Comment:
```suggestion
[TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) is a tool that collects binlog for TiDB clusters and provides near real-time sync and backup. It can be used for incremental data replication between TiDB clusters, such as making a TiDB cluster the secondary cluster of the primary TiDB cluster.
```